### ISSUE TITLE
CI for indicating that numba master does not break numba-dpex

### ISSUE THREAD

**Issue by PokhodenkoSA** [IntelPython/numba-dppy#619]
- [ ] GitHub workflow for testing stable numba-dpex with numba master
- [ ] Badge on main page
- [ ] Possibility to subscribe to the status for Numba team

**Comment by sklam** [comment]
In numba, we have been doing integration testing in this repo: https://github.com/numba/numba-integration-testing using CircleCI which is triggered on a CRON schedule. @esc made a tool, called [texasbbq](https://github.com/numba/texasbbq/blob/master/texasbbq.py), to simplify it. For example, the configuration for testing `umap` looks https://github.com/numba/numba-integration-testing/blob/fd58d5ce97357b0a2e4273fe4b89656370c8a2ed/switchboard.py#L30-L55. Under the hood, it is:

- git clone numba, install dependencies, and build it
- git clone `<downstream project>`, install dependencies, and build it
- run test in `<downstream project>`

**Comment by PokhodenkoSA** [comment]
@sklam @esc 
I think the right way is to prepare PR to https://github.com/numba/numba-integration-testing and add numba-dppy in the list. Ok?

**Comment by esc** [comment]
@PokhodenkoSA yes, indeed. The exitsing configurations will be a good place to start! Let me know if you need any assistence!

**Comment by PokhodenkoSA** [comment]
@esc
I am searching a way to test it locally before pushing PR.

**Comment by esc** [comment]
> @esc I am searching a way to test it locally before pushing PR. I will install `texasbbq` via pip and run `TARGET=numba-dppy ./switchboard.py -t $TARGET`.

That sounds right. You will also need the `packaging` package to make that work. But yes, in principle that is the way to run it locally. It should be documented, I hope?

**Comment by PokhodenkoSA** [comment]
I have created https://github.com/numba/numba-integration-testing/pull/81 for this.
